# Topic name for imu data
imu_data_topic_name: gps/imu

# Enable broadcasting of ROS transforms
broadcast_tfs: true

# Enable this to "plant" the origin of the map frame at the first received LLH
# position. If an existing map frame is defined, this should not be set.
create_map_frame: true

map_frame_name: map_veh

# Enable if you want to directly publish the measured earth -> gps TF
publish_earth_gpsm_tf: false
# The name of the measured GPS frame
measured_gps_frame: gps

# The frame name of the gps sensor, Novatel SPAN devices report data in the imu
# frame
static_gps_frame: gps

# Initialize without waiting for "INS_SOLUTION_GOOD" status from the Novatel
# Use at your own risk
no_solution_init: false

# Enable to use height above mean sea level instead of height above WGS84 ellipsoid
# Only works when mgrs_mode is true
msl_height: false

# MGRS mode is a simplified opertating mode that doesn't use ECEF and simply
# localizes in an existing map frame that presumably represents the nearest
# MGRS origin. The "create_map_frame" and "publish_earth_gpsm_tf" parameters
# have no effect if mgrs_mode is enabled.
# WARNING: if the MGRS zone changes during operation, the localizer will freeze
# in place, only use this mode if you are certain you will be operating in the
# same MGRS zone throughout.
mgrs_mode: false
